ini/features.ini

```diff
@@ -18,7 +18,7 @@ MARLIN_TEST_BUILD                      = src_filter=+<src/tests>
 POSTMORTEM_DEBUGGING                   = src_filter=+<src/HAL/shared/cpu_exception> +<src/HAL/shared/backtrace>
                                          build_flags=-funwind-tables
 MKS_WIFI_MODULE                        = QRCode=https://github.com/makerbase-mks/QRCode/archive/master.zip
-HAS_TRINAMIC_CONFIG                    = TMCStepper@~0.7.3
+HAS_TRINAMIC_CONFIG                    = TMCStepper=https://github.com/quiret/TMCStepper@~0.7.3
```

Comment:
It's important to get required changes into the upstream `TMCStepper` and tagged with a new version. We can use a fork on a temporary basis for testing, but the new `TMCStepper` version has to be published before Marlin is tagged with its own new version. You can submit your changes to the @teemuatlut fork as a PR if you haven't already done so. The current plan is to tag Marlin 2.1.2 this week, so this PR would be part of 2.1.3 after it has more testing behind it.

Reply:
I agree. The TMCStepper pull request is located at https://github.com/teemuatlut/TMCStepper/pull/255 which has been put up 11 days ago. I think that TMCStepper won't make 2.1.2 for sure, just looking at the frequency of updates in the official TMCStepper repository. Do you know of any way to reach the maintainer more privately?